app/services/imports/lettings_logs_import_service.rb

```diff
@@ -681,5 +682,17 @@ def is_carehome(scheme)
         0
       end
     end
+
+    def create_organisation_relationship(attributes)
+      parent_organisation_id = attributes["owning_organisation_id"]
+      child_organisation_id = attributes["managing_organisation_id"]
+      return if parent_organisation_id == child_organisation_id || organisation_relationship_exists?(parent_organisation_id, child_organisation_id)
+
+      OrganisationRelationship.create!(parent_organisation_id:, child_organisation_id:)
+    end
+
+    def organisation_relationship_exists?(parent_organisation_id, child_organisation_id)
+      OrganisationRelationship.find_by(parent_organisation_id:, child_organisation_id:).present?
+    end
```

Comment:
```suggestion

    def create_organisation_relationship!(attributes)
      parent_organisation_id = attributes["owning_organisation_id"]
      child_organisation_id = attributes["managing_organisation_id"]
      return if parent_organisation_id == child_organisation_id

      OrganisationRelationship.find_or_create_by!(parent_organisation_id:, child_organisation_id:)
    end
```

I think we can do this in one AR method.